src/hb-face.cc

```diff
@@ -178,10 +178,34 @@ hb_face_create (hb_blob_t    *blob,
   return face;
 }
 
+/**
+ * hb_face_get_count: Get number of faces on the blob
+ * @blob:
+ *
+ *
+ *
+ * Return value: Number of faces on the blob
+ *
+ * Since: REPLACEME
+ **/
+unsigned int
+hb_face_get_count (hb_blob_t *blob)
+{
+  if (unlikely (!blob))
+    return 0;
+
+  hb_blob_t *sanitized = OT::Sanitizer<OT::OpenTypeFontFile> ().sanitize (blob);
+  const char *base = hb_blob_get_data (sanitized, nullptr);
+  if (unlikely (!base))
+    return 0;
+
+  return OT::CastP<OT::OpenTypeFontFile> (base)->get_face_count ();
```

Comment:
No cast. Use the sanitizer LockInstance.

Reply:
Done